wandb/plot/pr_curve.py

```diff
@@ -176,6 +176,10 @@ def _step(x):
             "y": "precision",
             "class": "class",
         },
-        string_fields={"title": title},
+        string_fields={
+            "title": title,
+            "x-axis-title": "Recall",
+            "y-axis-title": "Precision",
```

Comment:
if you already add these, maybe we should let users the option to modify these fields?
same as the title

Reply:
not required! i see that we don't do it for the roc either